config/application.rb

```diff
@@ -9,7 +9,7 @@ class Application < Rails::Application
     config.encoding = "utf-8"
     config.filter_parameters += [:password]
     config.active_support.escape_html_entities_in_json = true
-
+    config.autoload_paths += %W["#{config.root}/app/validators/"]
```

Comment:
Whatever is under `/app/` directory is autoloaded to Rails. Also validator could be just in `lib`, but having it under app makes sense too :)